<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grap</class>
 <widget class="QMainWindow" name="grap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="chartView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>71</y>
      <width>421</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="light">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>光照</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grap_grp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="bar_Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>柱状图</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pie_Btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>饼状图</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="aver_lbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>平均数</string>
    </property>
   </widget>
   <widget class="QLabel" name="medium_lbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>中位数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beforeBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
